### PR TITLE
Organize rbs for `Lrama::Grammar::RuleBuilder`

### PR DESCRIPTION
**Pull request by ydah**

### CODE REVIEW

File: sig/lrama/grammar/rule_builder.rbs [hunk 1]
```diff
@@ -7,26 +7,35 @@ module Lrama
       attr_reader user_code: Lexer::Token::UserCode?
       attr_reader precedence_sym: Lexer::Token?
 
+      @rule_counter: Counter
+      @midrule_action_counter: Counter
+      @position_in_original_rule_rhs: Integer?
+      @skip_preprocess_references: bool?
```
Comment (yui-knk) [inline sig/lrama/grammar/rule_builder.rbs:13]: This is not optional, then `bool` (without `?`) is correct.

**Comment by yui-knk** [comment]
Left a 1 comment, other seems good.

File: sig/lrama/grammar/rule_builder.rbs (hunk 1 above)
Comment (ydah) [reply sig/lrama/grammar/rule_builder.rbs]: Indeed it is. I updated this PR/

**Comment by yui-knk** [comment]
👍

**Pull request merged**